examples/alb/index.ts

```diff
@@ -4,6 +4,8 @@ import * as elbv2 from 'aws-cdk-lib/aws-elasticloadbalancingv2';
 import * as pulumi from '@pulumi/pulumi';
 import * as pulumicdk from '@pulumi/cdk';
 
+const config = new pulumi.Config();
+const prefix = config.get('prefix') ?? 'local';
```

Comment:
This would still fail if two users executed the tests locally, right?
What about just generating a random string instead?

Reply:
I'll change it to `pulumi.getStack()`. It should be a stable value locally.